### PR TITLE
Rearrange variables

### PR DESCRIPTION
**Pull request by YounixM**
### Summary

Fixes https://github.com/SigNoz/signoz/issues/3802

https://www.loom.com/share/d9494bec594d4e439cb81a4c0476c06a?sid=ecae6e15-b623-4171-b0d5-fdb52bc02843

#### Affected Areas and Manually Tested Areas
Dashboard
Dashboard Variable Edit
Dashboard Variable Selection
Run Query
Test Run Query

### CODE REVIEW

**Comment by palashgdev** [comment]
@YounixM build is failing

**Comment by srikanthccv** [comment]
@YounixM every time you force-push my local branch diverges and results in conflicts making it hard to test. In general, It also makes it hard to follow what new changes have been made since the last comment. You can do regular pushes since we are going to squash them anyway.

**Review by srikanthccv — DISMISSED** [review]
Functionality wise LGTM

File: frontend/src/container/NewDashboard/DashboardVariablesSelection/DashboardVariableSelection.tsx [hunk 1]
```diff
@@ -21,8 +27,38 @@ function DashboardVariableSelection(): JSX.Element | null {
...
+					id,
+				});
+
+				if (order) {
+					variableOrderArr.push(order);
+				}
+			}
+
+			tableRowData.sort((a, b) => a.order - b.order);
+			variableOrderArr.sort((a, b) => a - b);
```
Comment (Rajat-Dabade) [inline frontend/src/container/NewDashboard/DashboardVariablesSelection/DashboardVariableSelection.tsx:56]: Do we need `variableOrderArr` as we are not using is further?

**Review by Rajat-Dabade — COMMENTED** [review]
One minor comment

File: frontend/src/container/NewDashboard/DashboardVariablesSelection/DashboardVariableSelection.tsx (hunk 1 above)
Comment (YounixM) [reply frontend/src/container/NewDashboard/DashboardVariablesSelection/DashboardVariableSelection.tsx]: @Rajat-Dabade : Addressed ✅

**Review by Rajat-Dabade — APPROVED** [review]
LGTM 🚀
